src/main/java/uk/gov/hmcts/darts/cases/service/impl/CloseOldCasesProcessorImpl.java

```diff
@@ -73,24 +84,34 @@ private void closeCase(CourtCaseEntity courtCase) {
             }
             if (!mediaList.isEmpty()) {
                 mediaList.sort(Comparator.comparing(MediaEntity::getCreatedDateTime).reversed());
-                closeCaseInDb(courtCase, mediaList.get(0).getCreatedDateTime());
+                closeCaseInDbAndAddRetention(courtCase, mediaList.get(0).getCreatedDateTime());
             } else {
                 //look for the last hearing date and use that
                 courtCase.getHearings().sort(Comparator.comparing(HearingEntity::getHearingDate).reversed());
                 HearingEntity lastHearingEntity = courtCase.getHearings().get(0);
-                closeCaseInDb(courtCase, OffsetDateTime.of(lastHearingEntity.getHearingDate().atStartOfDay(), ZoneOffset.UTC));
+                closeCaseInDbAndAddRetention(courtCase, OffsetDateTime.of(lastHearingEntity.getHearingDate().atStartOfDay(), ZoneOffset.UTC));
             }
         } else {
             //set to created date
-            closeCaseInDb(courtCase, courtCase.getCreatedDateTime());
+            closeCaseInDbAndAddRetention(courtCase, courtCase.getCreatedDateTime());
         }
 
 
     }
 
-    private void closeCaseInDb(CourtCaseEntity courtCase, OffsetDateTime caseClosedDate) {
+    private void closeCaseInDbAndAddRetention(CourtCaseEntity courtCase, OffsetDateTime caseClosedDate) {
         courtCase.setClosed(TRUE);
         courtCase.setCaseClosedTimestamp(caseClosedDate);
         caseRepository.save(courtCase);
+
+        LocalDate retentionDate = retentionDateHelper.getRetentionDateForPolicy(courtCase, RetentionPolicyEnum.DEFAULT);
+
+        PostRetentionRequest postRetentionRequest = new PostRetentionRequest();
+        postRetentionRequest.setComments("CloseOldCases Automated job setting retention period to Default");
```

Comment:
Maybe add the string as a class constant

Reply:
done